src/sinks/clickhouse.rs

```diff
@@ -659,6 +659,7 @@ timestamp_format = "unix""#,
     }
 
     #[derive(Debug, Deserialize)]
+    #[allow(dead_code)] // deserialize all fields
```

Comment:
I could be persuaded that we should just leave out the fields that we don't care about deserializing. I just thought it's useful to see what the full payload is.